src/ucp/core/ucp_am.c

```diff
@@ -320,6 +317,17 @@ ucp_am_fill_short_header(ucp_am_hdr_t *hdr, uint16_t id, uint16_t flags,
     hdr->header_length = header_length;
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_am_fill_iov_elem(uct_iov_t *iov, void *buffer, size_t length,
```

Comment:
something like ucp_add_uct_iov_elem - it's not am specific, and need "add" in the name because we're probably supposed to call it several times to add multiple elements
can we reuse it in other codes which fill iov? e.g zcopy